doc/user/content/sql/alter-source.md

```diff
@@ -20,10 +20,6 @@ menu:
 
 {{< diagram "alter-source-drop-clause.svg" >}}
 
-#### alter_source_set_clause
```

Comment:
This is covered in #26447, which would be a little cleaner in the commit history.

Reply:
Ah, it's not related to `ALTER ... SET CLUSTER`. It's related to legacy (pre-unified) source clusters. I failed to remove this piece in fdf70b7b9295b9192c08deffdffb07e4b064f312. I'll split it out into a separate commit here to keep the history cleaner.